firstvoices/backend/tests/test_apis/test_sites_api.py

```diff
@@ -941,3 +941,54 @@ def test_patch_success_200(self):
             data, self.get_updated_patch_instance(instance)
         )
         self.assert_update_patch_response(instance, data, response_data)
+
+    @pytest.mark.django_db
+    def test_hidden_sites_not_visible_in_list_view(self):
+        user = factories.get_app_admin(AppRole.SUPERADMIN)
+        self.client.force_authenticate(user=user)
+        factories.SiteFactory.create(visibility=Visibility.PUBLIC, is_hidden=True)
+
+        response = self.client.get(self.get_list_endpoint())
+        response_data = json.loads(response.content)
+        assert len(response_data) == 0
+        assert response_data == []
+
+    @pytest.mark.django_db
+    def test_hidden_site_detail_403(self):
```

Comment:
I think we need one more test like this one, but to prove that users with the MEMBER role also get a 403 on the detail api for a hidden site